packages/dapi/lib/chainDataProvider/ChainDataProvider.js

```diff
@@ -29,6 +32,11 @@ class ChainDataProvider extends EventEmitter {
    */
   blockHashHandler(blockHash) {
     this.emit(this.events.NEW_BLOCK_HEADER, blockHash.toString('hex'));
+    this.coreRpcAPI.getBestBlockHeight()
```

Comment:
It will once for a block for one DAPI process.

Reply:
yes